portfolio/src/main/webapp/myportfoliopage2.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+    <meta charset="utf-8">
+    <title>My Hobbies</title>
+    <link rel="stylesheet" href="style.css">
+    <body>
+        <div id="content">
+        <br><br><br>
+        <h1>My Hobbies</h1>
+
+        <h2>Games</h2>
+        <p> I'm more of a solo player and prefer story driven games, so here are my top 5 video games.<br>
+        The ones I listed here are some of the games that really impacted me with the experience that I got from them,<br>
+        and I think that if you're a gamer, you might find something you may want to try here!</p>
+       
+        <p> Resident Evil 2 Remake </p>
+        <a href="images/re2.jpg"><img src="images/re2.jpg" width="300" height="300" alt="Resident Evil 2 Remake"></a><
+        <p> Final Fantasy VII Remake (My #1 game)</p>
+        <a href="images/FF7R.jpg"><img src="images/FF7R.jpg" width="500" height="300" alt="Final Fantasy VII Remake"></a>
+        <p> Final Fantasy XIII Trilogy</p>
+        <a href="images/FF13.jpg"><img src="images/FF13.jpg" width="500" height="250" alt="Final Fantasy 13 Trilogy"></a>
+       <p> HALO</p>
```

Comment:
Indentation is just a bit off. Off by one space.

Reply:
Thank you for catching that! Just fixed the formatting.